src/bootstrap/check.rs

```diff
@@ -59,7 +59,7 @@ pub fn linkcheck(build: &Build, stage: u32, host: &str) {
     println!("Linkcheck stage{} ({})", stage, host);
     let compiler = Compiler::new(stage, host);
     build.run(build.tool_cmd(&compiler, "linkchecker")
-                   .arg(build.out.join(host).join("doc")));
+        .arg(build.out.join(host).join("doc")));
```

Comment:
This is the downside of using block indent rather than visual indent for method/field chains (cc https://github.com/rust-lang-nursery/fmt-rfcs/issues/8). Unfortunately, visual indent is a nightmare where there are closures nested within, and I don't think there is any way to combine the two.